pkg/config/otlp.go

```diff
@@ -87,3 +89,36 @@ func promoteExperimentalOTLP(cfg Config) {
 		cfg.Set(OTLPReceiverSection+".protocols.grpc.endpoint", net.JoinHostPort(getBindHost(cfg), v))
 	}
 }
+
+// setupEnvironmentVariables sets up the environment variables associated with different OTLP ingest settings:
+// If there are changes in the OTLP receiver configuration, they should be reflected here.
+//
+// We don't need to set the default value: it is dealt with at the unmarshaling level
+// since we get the configuration through GetStringMap
+//
+// We are missing TLS settings: since some of them need more work to work right they are not included here.
+func setupEnvironmentVariables(config Config) {
```

Comment:
Should probably have `otlp` in the name somewhere, because the func is in the common `config` package.

Reply:
done!